### ISSUE TITLE
Unable to Delete folder on Windows (0.1.17)

### ISSUE THREAD

**Issue by blakev** [Librevault/librevault#42]
After using the "Add folder" button to track a generic folder, using the "Delete" button does not remove it from the list.

The item persists between application restarts, so it's not just a UI problem.

**Comment by KrisZane** [comment]
@blakev I too had this problem, I found that if you delete the .librevault folder, it will remove the folder. My librevault kind of broke after that, so not entirely sure if they are related. :/

**Comment by rasb23** [comment]
Adding to this due to GamePad64 comment on:
https://forum.librevault.com/t/how-to-get-started/60/2

When adding a folder I have no issue but a folder does not show up in the main panel. Log shows:
==>  QJsonDocument({"command":"add_folder","folder":{"path":"/share/photos","secret":"password"}}

after a while I see:
Disconnected from daemon:  QUrl("ws://localhost:42346")  Reason:  ""

This is on ubuntu-mate 16.04 running libraVault 0.1.18

**Comment by rikai** [comment]
Yeah, I can also confirm this... I added several test folders and now when i delete them, they just stay there anyway.

If there's anything i can do to help further debug this, I'm happy to help!
